README.md

```diff
@@ -2,14 +2,17 @@
 Elasticsearch plugin which only provides a TokenFilter that merges tokens in a token stream back into one. Taken from http://elasticsearch-users.115913.n3.nabble.com/Is-there-a-concatenation-filter-td3711094.html
 
 ## ElasticSearch version support
-This plugin targets 1.X versions of ES, and won't work for 2.X.
+This plugin is compatible with ES 5.4.2. The original forked version is at https://github.com/francesconero/elasticsearch-concatenate-token-filter
```

Comment:
We plan to create a merge request against that fork, therefore, that extra info will not be necessary anymore. No?

Reply:
Make sense, I have removed the unnecessary reference.